src/app/parcels/ParcelsPage.cy.tsx

```diff
@@ -162,25 +126,6 @@ describe("Parcels Page", () => {
     });
 
     describe("Backend Processing for Expanded Parcel Details", () => {
-        it("Fields are set correctly", () => {
-            const expandedClientDetails = rawDataToExpandedParcelData(
```

Comment:
This test is no longer relevant

Reply:
is this because the data is got from the parcels_plus view?

Reply:
This test specifically tests `rawDataToExpandedParcelData`, which we don't have anymore. (Though we could spend some effort to stub out the supabase calls to test effectively the same thing, but probably not worth the effort for now)